lib/pbio/platform/ev3/platform.ld

```diff
@@ -19,32 +19,42 @@ SECTIONS
     {
         . = ALIGN(4);
         *(.text)
+        *(.text.*)
     } > DDR
 
+    .rodata :
+    {
+        . = ALIGN(4);
+        *(.rodata)
+        *(.rodata.*)
+    } > DDR
+
+    .pru :
```

Comment:
Since we are moving this anyway, we could rename it to `.firmware` since will we have at least Bluetooth firmware to add to here as well.